src/utils/functions.ts

```diff
@@ -0,0 +1,10 @@
+
+
+export function getFormattedDate(sendtDate: string){
```

Comment:
Tror denne kan erstattes av `new Date(sendDate!).toLocaleString('nb-NO')`. 
Dokumentasjon her; https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

Reply:
Prøvde meg på dette. Eneste problemet der var at det kom et komma mellom dato og tid. Unngikk dette ved å bruke `.toLocalDateString` og `.toLocalTimeString` og slå de strengene sammen. Dersom det er en snedigere måte å unngå kommaet på tar jeg gjerne i mot forslag.

Reply:
Det er nok da egentlig den korrekte måten å skrive dato og tid på norsk. 🤨
Fant ingenting i dokumentasjonen som tilsier at man kan endre på akkurat den biten dessverre. Så må man nok slå sammen de to andre kallene, eller evt bruke ett bibliotek som har mulighet til å formatere for oss.

Reply:
Ja, det stemmer nok at det er den korrekte måten... Kan spørre designerene - selv synes jeg det var finest med mellomrom. Prøver meg på Alexander sitt forslag med å bruke `date-fns`